### DISCUSSION TITLE
Hitsplat API

### DISCUSSION THREAD

**Discussion by Mark7625** [runelite/runelite#17089] [Ideas]
Hey so I have recently been working on some improvements to the interface styles 

![image](https://github.com/runelite/runelite/assets/72366279/7111d83b-3d79-4ca9-a1d2-a80d87ab3d14)

But when I came to adding hitsplats they only override on login, I looked into a deob and it seems like they are cached and only cleared on a logout, This makes sense as if I log in , it then updates

I also found out that Jagex has support to animate these built into the client by adding a few configs so if these could be exposed would be fantastic here is a list of things I request. 

- Method to reset the cache of the hitsplats
- Exposed methods to get and set opcode [11,10,9] or all if possible 
- PostHitSplat event so i can set the data

**Comment by SirFancyBacon** [comment]
I second this; I would love to have better hitsplats; i mostly just want the block from 2010.

**Comment by Jin-Jiyunsun** [comment]
+1 from me, would be an awesome addition

**Comment by NicolitoP** [comment]
I totally support it!

**Comment by hunter2451** [comment]
Posting here for support, I would love to see this added if possible.
